client/src/common/components/Members/components/MemberDetails.jsx

```diff
@@ -282,12 +279,15 @@ class MemberDetails extends PureComponent {
           </button>
           <div className="member-details__details">
             {this.renderHeader()}
-            {isCurrentUserAnOwner &&
-              userId !== currentUserId &&
-              this.renderDetails()}
-            {isCurrentUserAnOwner &&
-              userId === currentUserId &&
-              this.renderMessage()}
+            <div className="member-details__panel">
+              {isCurrentUserAnOwner &&
+                userId !== currentUserId &&
+                this.renderDetails()}
```

Comment:
if current user is an Owner and enters own memerBox (with own details) he shouldn't delete yourself from cohort/list or edit own permissions here. This if statements render special info for such scenario and disable rendering ui for user management.
It can be write in this way:
```
{isCurrentUserAnOwner && (
  <Fragment>
    {userId !== currentUserId
      ? this.renderDetails()
      : this.renderMessage()}
    </Fragment>
)}
```
If you think this more clearly I can change it.

Reply:
yes please change it, 
and do not click "resolved" for comments where there are some responses to read.